exonum-java-binding-qa-service/src/test/java/com/exonum/binding/qaservice/QaServiceImplIntegrationTest.java

```diff
@@ -308,6 +315,22 @@ void getBlockTransactions() {
     }
   }
 
+  @Test
+  void afterCommit() throws CloseFailuresException {
+    try (MemoryDb db = MemoryDb.newInstance();
+         Cleaner cleaner = new Cleaner()) {
+      Fork fork = db.createFork(cleaner);
+      setServiceNode(node);
+      service.initialize(fork);
+
+      Snapshot snapshot = db.createSnapshot(cleaner);
+      BlockCommittedEvent event = BlockCommittedEventImpl.valueOf(snapshot, OptionalInt.of(1), 1L);
+      service.afterCommit(event);
+
+      verify(service).submitIncrementCounter(anyLong(), any(HashCode.class));
```

Comment:
Such use of Mockito won't work — it can only verify invocations on spies and mocks. Shan;t you verify invocation of a `Node#submitTransaction` with a certain transaction?